docs/spec_draft.md

```diff
@@ -162,6 +162,7 @@ described below)
    * [logistic](#stablehlologistic)
    * [maximum](#stablehlomaximum)
    * [minimum](#stablehlominimum)
+   * [mul](#stablehlomultiply)
```

Comment:
Let's change this to use mnemonic `multiply`:
`* [multiply](#stablehlomultiply)`